apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java

```diff
@@ -142,6 +146,30 @@ protected void onCreate(Bundle savedInstanceState) {
             Push.enableFirebaseAnalytics(this);
         }
 
+        /* Set max storage size. */
+        final long maxStorageSize = sSharedPreferences.getLong(MAX_STORAGE_SIZE_KEY, 0);
+        if (maxStorageSize > 0) {
+            AppCenter.setMaxStorageSize(maxStorageSize).thenAccept(new AppCenterConsumer<Boolean>() {
+                @Override
+                public void accept(Boolean aBoolean) {
+                    if (aBoolean) {
+                        long expectedMultipleMaxSize = maxStorageSize / 4096 * 4096;
```

Comment:
If you want to match the exact logic we're using in the SDK, I updated it to use the `ceil()` function: https://github.com/Microsoft/AppCenter-SDK-Android/pull/796/files#diff-9b79a28544bf85990330f7945382e4ebR523

This also saves you the three lines below